### DISCUSSION TITLE
Is making a PR for adding many domains to the global dark list allowed ?

### DISCUSSION THREAD

**Discussion by KaKi87** [darkreader/darkreader#10701] [Q&A] [answered]
Hello,

As I'm going to clean up my *not invert list* today, I'll have a list of several domains to add to the global dark list (after carefully checking compliance with [the requirements](https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md#adding-a-website-that-is-already-dark), of course).

So, I wonder, will I be allowed to make a PR containing adding multiple domains at once ?

Thanks

**Comment by Gusted** [comment, marked as answer]
Yes, that's fine!
